generator-mediator-js/app/templates/packagingTemplate/targets/trusty/etc/init/_mediator.conf

```diff
@@ -0,0 +1,20 @@
+# <%= mediatorName %>
```

Comment:
This file will need to be named app-name.conf to work properly with start and stop commands in `_postinst`.